### PR TITLE
Add E2E testing for jenkins spec

### PR DESCRIPTION
**Pull request by cescgina**
Add a few e2e test focused on the spec subcommand for the jenkins source

### CODE REVIEW

File: tests/cibyl/e2e/test_jenkins.py [hunk 1]
```diff
@@ -681,3 +682,206 @@ def test_filter_ip_show_last_build(self):
...
+        self.assertIn("No query performed", self.stdout)
+
+    def test_get_spec_multiple_jobs(self):
+        """Check the spec command fails if requesting the spec for many
+        jobs."""
+        sys.argv = [
+            'cibyl',
+            '--config', 'tests/cibyl/e2e/data/configs/jenkins.yaml',
+            '-p', 'openstack',
+            '-f', 'text', 'spec', 'test'
```
Comment (adrianfusco) [inline tests/cibyl/e2e/test_jenkins.py:883]: In this case which are the multiple jobs?

Comment (cescgina) [reply tests/cibyl/e2e/test_jenkins.py:883]: There are several jobs in the e2e testing environment that would match the pattern `test` (test_1, test_2, test_3, ...)

Comment (adrianfusco) [reply tests/cibyl/e2e/test_jenkins.py:883]: That's true.

**Review by adrianfusco — APPROVED** [review]
Nice from me side +1.

**Pull request merged**